docs/content/guide/bootstrap.ngdoc

```diff
@@ -142,6 +142,17 @@ This is the sequence that your code should follow:
   2. Call {@link angular.bootstrap} to {@link compiler compile} the element into an
   executable, bi-directionally bound application.
 
+## Things to keep in mind
+
+There a few things to keep in mind regardless of automatic or manual bootstrapping:
+
+- There is no explicit support for bootstrapping more than one AngularJS applications on a page, and
+  doing this may cause bugs and/or memory leaks.
+- Likewise, apps cannot be nested within each other.
```

Comment:
Oh, interesting. I copied this from ngApp, so it apparently only applies to ngApp, not manual bootstrapping